setup.py

```diff
@@ -17,14 +17,20 @@ def readme():
         'numba>=0.50.0, !=0.54.0, <0.60.0',  # Avoid 0.54 due to: https://github.com/SeldonIO/alibi/issues/466
     ],
 
-    'tensorflow': ['tensorflow>=2.0.0, !=2.6.0, !=2.6.1, <2.15.0'],
+    # `keras 3` becomes the default for `tensorflow >= 2.16.0``
+    # which is not yet supported by `transformers`
+    'tensorflow': [
+        'tensorflow>=2.0.0, !=2.6.0, !=2.6.1, <2.19.0',
```

Comment:
should the min version for tf be 2.16.0 now?

Reply:
Will do in 3.12 PR